src/agenteval/cli.py

```diff
@@ -120,146 +128,98 @@ def cli():
     help="Score a directory of evaluation logs.",
 )
 @click.argument("log_dir", type=click.Path(exists=True, file_okay=False))
-@click.option(
-    "--config-path",
-    "config_path",
-    type=str,
-    help=f"Path to a yml config file. Ignored if {EVAL_FILENAME} exists.",
-    default=None,
-)
-@click.option(
-    "--split",
-    type=str,
-    help=f"Config data split. Ignored if {EVAL_FILENAME} exists.",
-    default=None,
-)
 def score_command(
     log_dir: str,
-    config_path: str | None,
-    split: str | None,
```

Comment:
How would one score/publish a single-task eval now? (I didn't see a clear path for it in the RFC doc, and it doesn't look like we're getting `astabench eval --task` or `--task-category` in this PR)

Reply:
Could we maybe handle this by defining single-task splits? That seems fairly natural. We'd end up with separate leaderboards for each of them.

Reply:
(here's the [ticket](https://github.com/allenai/astabench-issues/issues/78) with the --task proposal)